src/Sylius/Bundle/AdminApiBundle/Migrations/Version20210311142134.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminApiBundle\Migrations;
```

Comment:
@pamil I think this is a wrong namespace (automatically generated).
Sylius\Bundle\UserBundle\Migrations ?

Reply:
Oh, that's right! Let's move it to CoreBundle, we only provide migrations there.